ykcompile/src/lib.rs

```diff
@@ -232,8 +236,8 @@ impl<TT> TraceCompiler<TT> {
                 // making this operation more complicated and costly.
                 dynasm!(self.asm
                     ; mov rax, [rbp - off2]
-                    ; mov [rbp - off1], rax
                 );
+                asm_mem_reg!(&mut self.asm, SIZE_ALL, mov, [rbp - off1], RAX.code());
```

Comment:
Maybe we should just pass `RAX` here. Thought for that we need to refactor a bit of code, so maybe we do a refactoring PR at some later point.

Reply:
It may or may not be possible. The macro is expecting something which can go inside a `Rq()`.

Reply:
Can't we just say `$reg.code()` inside the macro?

Reply:
I'll investigate.

Reply:
If we put `.code()` inside the macro, then we cannot use it with `Location::Register()` which stores the raw register number already.